rules.js

```diff
@@ -189,10 +224,17 @@ module.exports = function(styles, opts={}) {
           }, output(node.content, state));
           state.withinParagraphWithImage = false
           return paragraph
+        } else if (_.size(node.content) < 3 && _.some(node.content, {type: 'strong'})){
```

Comment:
the `parse` is not working properly as it should, the whole paragraph of `strong` type will have a `text` object of empty string, yeah, i will put a comment on this